### ISSUE TITLE
Version redis-stack 6.2.2-5 not working

### ISSUE THREAD

**Issue by Glennmen** [redis-stack/homebrew-redis-stack#4]
After updating redis-stack to the latest version redis-stack-server isn't able to start anymore.

```
32676:M 05 Jul 2022 11:32:56.708 # Server initialized
32676:M 05 Jul 2022 11:32:56.714 * <search> Redis version found by RedisSearch : 6.2.7 - oss
32676:M 05 Jul 2022 11:32:56.714 * <search> RediSearch version 2.4.9 (Git=HEAD-55607bc4)
32676:M 05 Jul 2022 11:32:56.714 * <search> Low level api version 1 initialized successfully
32676:M 05 Jul 2022 11:32:56.715 * <search> concurrent writes: OFF, gc: ON, prefix min length: 2, prefix max expansions: 200, query timeout (ms): 500, timeout policy: return, cursor read size: 1000, cursor max idle (ms): 300000, max doctable size: 1000000, max number of search results:  10000, search pool size: 20, index pool size: 8, 
32676:M 05 Jul 2022 11:32:56.715 * <search> Initialized thread pool!
32676:M 05 Jul 2022 11:32:56.715 * <search> Enabled diskless replication
32676:M 05 Jul 2022 11:32:56.715 * Module 'search' loaded from /opt/homebrew/Caskroom/redis-stack-server/6.2.2-v5/lib/redisearch.so
32676:M 05 Jul 2022 11:32:56.722 * <graph> Starting up RedisGraph version 2.8.15.
32676:M 05 Jul 2022 11:32:56.722 * <graph> Thread pool created, using 10 threads.
32676:M 05 Jul 2022 11:32:56.722 * <graph> Maximum number of OpenMP threads set to 10
32676:M 05 Jul 2022 11:32:56.722 * Module 'graph' loaded from /opt/homebrew/Caskroom/redis-stack-server/6.2.2-v5/lib/redisgraph.so
32676:M 05 Jul 2022 11:32:56.723 # Module /opt/homebrew/Caskroom/redis-stack-server/6.2.2-v5/lib/redistimeseries.so failed to load: dlopen(/opt/homebrew/Caskroom/redis-stack-server/6.2.2-v5/lib/redistimeseries.so, 0x0006): Library not loaded: /opt/homebrew/opt/gcc/lib/gcc/11/libgcc_s.1.1.dylib
  Referenced from: /opt/homebrew/Caskroom/redis-stack-server/6.2.2-v5/lib/redistimeseries.so
  Reason: tried: '/opt/homebrew/opt/gcc/lib/gcc/11/libgcc_s.1.1.dylib' (no such file), '/usr/lib/libgcc_s.1.1.dylib' (no such file)
32676:M 05 Jul 2022 11:32:56.723 # Can't load module from /opt/homebrew/Caskroom/redis-stack-server/6.2.2-v5/lib/redistimeseries.so: server aborting
```

Seems like a missing dependency, isn't this something that the brew script should install if it is needed?

**Comment by chayim** [comment]
@Glennmen can you please share - which version of redis-stack were you upgrading from, your version of OSX, and also are you architecture (Intel vs M1)?

**Comment by Glennmen** [comment]
I believe it was 6.2.2-v4
MacOS Monterey 12.4
M1 Pro

**Comment by RicardoValero95** [comment]
Any luck on this?
I can't downgrade to the previous version

**Comment by Glennmen** [comment]
@RicardoValero95 I installed the missing dependency manually and that solved it for me.
`brew install gcc`

**Comment by chayim** [comment]
Thanks @Glennmen. There was an issue with how the underling RTS module was built. It's actually not a redis-stack issue. I'm keeping this open as I'd like to ensure this is solved in the next release. I don't want gcc to be a dependency, given that there is a downloadable version of redis-stack (though yes, it requires dependencies).

**Comment by chayim** [comment]
@Glennmen we released the first in the 6.2.4 family which removes this dependency. I'm going to mark this bug as closed now - but I wanted to give you the heads up.  Here's the output of otool - in case you're interesed:

```
otool -L /opt/homebrew/Caskroom/redis-stack-server/6.2.4-v1/lib/*
/opt/homebrew/Caskroom/redis-stack-server/6.2.4-v1/lib/redisbloom.so:
	/w/rafi_1/RedisBloom-2.2/redisbloom.so (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
/opt/homebrew/Caskroom/redis-stack-server/6.2.4-v1/lib/redisearch.so:
	/w/rafi_1/RediSearch-2.4/bin/macos-arm64v8-release/search/redisearch.so (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
/opt/homebrew/Caskroom/redis-stack-server/6.2.4-v1/lib/redisgraph.so:
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
	/opt/homebrew/opt/libomp/lib/libomp.dylib (compatibility version 5.0.0, current version 5.0.0)
/opt/homebrew/Caskroom/redis-stack-server/6.2.4-v1/lib/redistimeseries.so:
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
	/opt/homebrew/opt/openssl@3/lib/libssl.3.dylib (compatibility version 3.0.0, current version 3.0.0)
	/opt/homebrew/opt/openssl@3/lib/libcrypto.3.dylib (compatibility version 3.0.0, current version 3.0.0)
/opt/homebrew/Caskroom/redis-stack-server/6.2.4-v1/lib/rejson.so:
	/w/rafi_1/RedisJSON-2.2/bin/macos-arm64v8-release/target/release/deps/librejson.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
	/usr/lib/libresolv.9.dylib (compatibility version 1.0.0, current version 1.0.0)
```
